index.html

```diff
@@ -11175,13 +11175,17 @@ <h2>Definitions of States and Properties (all aria-* attributes)</h2>
 		<div class="property" id="aria-controls">
 			<pdef>aria-controls</pdef>
 			<div class="property-description">
-				<p><a>Identifies</a> the <a>element</a> (or elements) whose contents or presence are controlled by the current element. See related <pref>aria-owns</pref>.</p>
-				<p>For example:</p>
+				<p><a>Identifies</a> the <a>element</a> (or elements) whose contents or presence are controlled by the focused element or composite widget. See related <pref>aria-details</pref>.</p>
```

Comment:
This implies that the element must be focused in order to have aria-controls on it

Reply:
```suggestion
				<p><a>Identifies</a> the <a>element</a> (or elements) whose contents or presence are controlled by an element or composite widget. See related <pref>aria-details</pref>.</p>
```

(though admittedly this might make it more difficult to understand

Reply:
using 'focused' was intential here.  i'm not sure i'd want to change that.